public/static-checkout/unminified.js

```diff
@@ -169,6 +200,12 @@ document.addEventListener('DOMContentLoaded', function () {
   renderCountryState();
 });
 
+/* eslint-disable no-unused-vars */
+window.onload = function sendPageLoaded(event) {
+  sendRev1074Event('static.page_loaded', {});
```

Comment:
is this the only event we need?

Reply:
No, there is at least a couple of others (see REV-1116)